### DISCUSSION TITLE
The `readme.md` needs to be updated.

### DISCUSSION THREAD

**Discussion by ImAQwertyGuy** [Exarkuniv/RetroPie-Extra#222] [General]
As you can tell by the title of this issue, `readme.md` needs to be updated, as I could not figure out what `srbkart2.sh` is because it's not in the readme.

**Comment by Exarkuniv** [comment]
Yes I have been meaning to get that. And I will
But I have other things I have been working on

**Comment by Gemba** [comment]
@ImAQwertyGuy After SSH'ing to your Pi:  `sudo RetroPie-Setup/retropie_packages.sh | grep srb2kart`
